From eca5131cbbc7e1785b266cc92624a2ce9582a7cc Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 1 Aug 2023 12:42:05 +0100
Subject: [PATCH] tests/test-ocaml-errorcodes.c: Enable verbose messages

This test crashes when the OCaml plugin is unloaded (specifically when
we call caml_acquire_runtime_system).  Enable verbose messages in a
failed attempt to track down what is happening.

(cherry picked from commit 99567408fdd8ea55b0bb5286b45ce135d91c38b5)
---
 tests/test-ocaml-errorcodes.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test-ocaml-errorcodes.c b/tests/test-ocaml-errorcodes.c
index 8ab7e0ae..f84da3d8 100644
--- a/tests/test-ocaml-errorcodes.c
+++ b/tests/test-ocaml-errorcodes.c
@@ -78,7 +78,7 @@ main (int argc, char *argv[])
 
   if (nbd_connect_command (nbd,
                            (char *[]) {
-                             "nbdkit", "-s", "--exit-with-parent",
+                             "nbdkit", "-s", "--exit-with-parent", "-v",
                              "./test-ocaml-errorcodes-plugin.so",
                              NULL }) == -1) {
     fprintf (stderr, "%s\n", nbd_get_error ());
-- 
2.39.3

